.codex/skills/zammad-mcp-issue-digest/scripts/triage_common.py

```
@@ -0,0 +1,678 @@
+            out["duplicate_candidates"] = self.duplicate_candidates
+        else:
+            out["duplicate_candidates"] = []
+        if self.state_summary is not None:
+            out["state_summary"] = self.state_summary
+        return out
+
+
+def run_gh(args: list[str], *, input_text: str | None = None, check: bool = True) -> subprocess.CompletedProcess:
+    cmd = ["gh", *args]
+    try:
+        proc = subprocess.run(  # nosec B603 B607
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)